examples/simulation_pytorch/simulation.py

```diff
@@ -0,0 +1,121 @@
+
+        def set_parameters(self, parameters):
+            params_dict = zip(net.state_dict().keys(), parameters)
+            state_dict = OrderedDict({k: torch.Tensor(v) for k, v in params_dict})
+            net.load_state_dict(state_dict, strict=True)
+
+        def fit(self, parameters, config):
+            self.set_parameters(parameters)
+            train(net, trainloader, epochs=1)
+            return self.get_parameters(), len(trainloader), {}
+
+        def evaluate(self, parameters, config):
+            self.set_parameters(parameters)
+            loss, accuracy = test(net, testloader)
+            return float(loss), len(testloader), {"accuracy": float(accuracy)}
+
+    # Start Flower client
+    fl.client.start_numpy_client("0.0.0.0:8080", client=CifarClient())
+
+
+def train(net, trainloader, epochs):
+    """Train the network on the training set."""
+    criterion = torch.nn.CrossEntropyLoss()
+    optimizer = torch.optim.SGD(net.parameters(), lr=0.001, momentum=0.9)
+    net.train()
+    for _ in range(epochs):
+        for images, labels in trainloader:
+            images, labels = images.to(DEVICE), labels.to(DEVICE)
+            optimizer.zero_grad()
+            loss = criterion(net(images), labels)
+            loss.backward()
+            optimizer.step()
+
+
+def test(net, testloader):
+    """Validate the network on the entire test set."""
+    criterion = torch.nn.CrossEntropyLoss()
+    correct, total, loss = 0, 0, 0.0
+    net.eval()
+    with torch.no_grad():
+        for data in testloader:
+            images, labels = data[0].to(DEVICE), data[1].to(DEVICE)
+            outputs = net(images)
+            loss += criterion(outputs, labels).item()
+            _, predicted = torch.max(outputs.data, 1)
+            total += labels.size(0)
+            correct += (predicted == labels).sum().item()
+    accuracy = correct / total
+    return loss, accuracy
+
+
+def run_simulation(num_rounds: int, num_clients: int, fraction_fit: float):
+    """Start a FL simulation."""
+    # This will hold all the processes which we are going to create
+    processes = []
+
+    # Start the server
+    server_process = Process(
+        target=start_server, args=(num_rounds, num_clients, fraction_fit)
+    )
+    server_process.start()
+    processes.append(server_process)
+
+    # Optionally block the script here for a second or two so the server has time to start
+    time.sleep(2)
+
+    # Load the dataset partitions
+    partitions = dataset.load(num_clients)
+
+    # Start all the clients
+    for partition in partitions:
+        client_process = Process(target=start_client, args=(partition,))
+        client_process.start()
+        processes.append(client_process)
+
+    # Block until all processes are finished
+    for p in processes:
+        p.join()
+
+
+if __name__ == "__main__":
+    torch.multiprocessing.set_start_method("spawn") 
+    run_simulation(num_rounds=1, num_clients=10, fraction_fit=1)
```

Comment:
This is not in sync with the README.md. So I would suggest:

```suggestion
    run_simulation(num_rounds=100, num_clients=10, fraction_fit=0.5)
```